kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js

```diff
@@ -0,0 +1,80 @@
+define(['bluebird', 'jquery', 'common/html', 'common/ui', 'common/events'], function (
+    Promise,
+    $,
+    html,
+    UI,
+    Events
+) {
+    'use strict';
+
+    let tag = html.tag,
+        div = tag('div');
+
+    function OutputWidget() {
+        let container, ui;
+
+        function renderOutput(data) {
+            return new Promise((resolve) => {
+                console.log('ready to render object output with data: ', data);
```

Comment:
What is the purpose of this function?

Reply:
this is just a placeholder for where we can render the output. i just put it in there to make sure the plumbing was hooked up in the way i expected